File: java/src/main/java/model/params/CandleRawParams.java
package model.params;

import model.ChannelTypes;

public class CandleRawParams extends Params {

    private String interval;

    public CandleRawParams(String symbol, String interval) {
        super(ChannelTypes.CANDLERAW, symbol);
        this.interval = interval;
    }

    public String getInterval() {
        return interval;
    }

    public void setInterval(String interval) {
        this.interval = interval;
    }

    @Override
    public String toString() {
        return "CandleRawParams{" +
                super.toString() +
                ", interval=" + interval +
                "} ";
    }
}

File: c-sharp/ws-client/model/parameters/CandleRawParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_client.model.parameters
{
    class CandleRawParams : Params
    {
        public CandleRawParams(string symbol, string interval) : base(ChannelTypes.CANDLE, symbol)
        {
            this.interval = interval;
        }

        public string interval { get; set; }
    }
}
